src/components/macroview/centerPanel/sortableElement/SortableItem.tsx

```diff
@@ -101,13 +96,13 @@ export default function SortableItem({
     if (checkIfElementIsEditable(element)) updateSelectedElementId(id - 1)
   }, [element, id, isSelected, recording, updateSelectedElementId])
 
-  const onEditButtonPress = useCallback(() => {
-    stopRecording()
-    if (isSelected) {
-      return
-    }
-    updateSelectedElementId(id - 1)
-  }, [id, isSelected, stopRecording, updateSelectedElementId])
+  // const onEditButtonPress = useCallback(() => {
+  //   stopRecording()
+  //   if (isSelected) {
+  //     return
+  //   }
+  //   updateSelectedElementId(id - 1)
+  // }, [id, isSelected, stopRecording, updateSelectedElementId])
```

Comment:
Same here, need to keep this if the button was removed?

Reply:
Fixed